externs/browser/w3c_audio.js

```diff
@@ -26,33 +26,64 @@
 /**
  * @constructor
  */
-function AudioContext() {}
+function BaseAudioContext() {};
 
-/** @type {!AudioDestinationNode} */
-AudioContext.prototype.destination;
+/**
+ * @type {!AudioDestinationNode}
+ * @readonly
+ */
+BaseAudioContext.prototype.destination;
 
-/** @type {number} */
-AudioContext.prototype.sampleRate;
+/**
+ * @type {number}
+ * @readonly
+ */
+BaseAudioContext.prototype.sampleRate;
 
-/** @type {number} */
-AudioContext.prototype.currentTime;
+/**
+ * @type {number}
+ * @readonly
+ */
+BaseAudioContext.prototype.currentTime;
 
-/** @type {!AudioListener} */
-AudioContext.prototype.listener;
+/**
+ * @type {!AudioListener}
+ * @readonly
+ */
+BaseAudioContext.prototype.listener;
 
 /**
+ * @readonly
```

Comment:
`@readonly` is not an annotation that closure-compiler understands.
https://github.com/google/closure-compiler/blob/master/src/com/google/javascript/jscomp/parsing/Annotation.java

Please drop it, since including it here misleads the reader into thinking it means something.